.circleci/real_config.yml

```diff
@@ -1942,19 +1942,59 @@ jobs:
             - webui/react/build
 
   test-e2e-react:
-    docker:
-      - image: mcr.microsoft.com/playwright:v1.40.0-focal
+    machine:
+      image: <<pipeline.parameters.machine-image>>
     resource_class: large
     steps:
+      - run: echo 'export PW_SERVER_ADDRESS="http://localhost:3001"' >> $BASH_ENV # DNJ TODO -remove when tested
+      - run: echo 'export PW_USER_NAME=admin' >> $BASH_ENV
+      - run: echo 'export PW_PASSWORD=' >> $BASH_ENV
+      - run: echo 'export DET_WEBPACK_PROXY_URL="http://localhost:8082"' >> $BASH_ENV
+      - run: echo 'export DET_WEBSOCKET_PROXY_URL="ws://localhost:8082"' >> $BASH_ENV
       - checkout-with-sm
       - add-and-fetch-upstream
       - skip-if-only-docs
       - skip-if-only-github
-      - run: apt-get update && apt-get -y install make
+      - attach_workspace:
+          at: .
       - react-get-deps
+      - run: |
+          apt-get update && sudo apt-get install -y ca-certificates curl make sudo python3-pip iproute2 tar
+          # Docker repo setup DNJ TODO - move to own step and vet all deps
+          sudo install -m 0755 -d /etc/apt/keyrings
+          sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+          sudo chmod a+r /etc/apt/keyrings/docker.asc
+          echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+            $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+            sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+          apt-get update && sudo apt-get -y install docker-ce docker-ce-cli containerd.io
+          sudo systemctl start docker
```

Comment:
I suspect it was from the `container_name` on the postgres config. I assume that starts postgres in docker. I'm trying with that removed. Is this config documented somewhere? I looked in devcluster and didn't see anything that stuck out in the help or readme

Reply:
we have a bunch of jobs in `real_config.yaml` which run `install-devcluster` and `start-devcluster`. they use the same `machine-image`, and they don't setup docker.

e.g. here's one which runs db stage only so it'll need docker: https://github.com/determined-ai/determined/blob/main/.circleci/real_config.yml#L1905

Reply:
I remember now. In the initial iteration we used the official playwright image, but with devcluster this leads to a docker-in-docker pain-scenario. I switched to using our default vm image, using standard dependency steps, and just installing playwright after. I deleted this step entirely since it shouldn't be necessary anymore. I'll update this comment if it seems like any additional dependencies are needed.